### ISSUE TITLE
Regression: Message "Search field empty, please enter your search query." as result of search execution.

### ISSUE THREAD

**Issue by sabina-talipova** [silverstripe/cwp-search#58]
## Description
Regression tests have been executed on: SC PHP 7.4 Chrome GraphQL v3 Sink.
When user is searching for any word, he gets "Search field empty, please enter your search query.".

Probably, related to this issue:
- https://github.com/silverstripeltd/product-issues/issues/709

https://studio.cucumber.io/projects/301855/test-runs/758277/folder-snapshots/8007994/scenario-snapshots/28020701/test-snapshots/38195476

## AC's
- Validate why the Solr search is not working on SC
- 

## PR

**Comment by maxime-rainville** [comment]
I follow up with the Service desk on this. ~~I don't think the problem is with the module.~~

**Comment by maxime-rainville** [comment]
I'm getting the error on SC and SCPS. This seems a bit too far ranging for comfort.

**Comment by sabina-talipova** [comment]
I have redeployed CMS 4.12 and done additional tests for this issue. It looks like this issue is unrelated to our code base and probably exist due some issues in the configuration of our SC and SCPS platforms.
@maxime-rainville should discuss this issue with our support team.
Waiting response.

**Comment by maxime-rainville** [comment]
Looks like we might have a problem with the `stopwords.txt` file not being copied to the relevant folder.

Looks like this [file and others](https://github.com/silverstripe/silverstripe-fulltextsearch/blob/3/docs/en/03_configuration.md#file-based-configuration) are meant to be copied to the project folder:

> ## File-based configuration
> 
> Many aspects of Solr are configured outside of the `schema.xml` file which SilverStripe generates based on the `SolrIndex` subclass that is defined. For example, stopwords are placed in their own `stopwords.txt` file, and advanced [spellchecking](05_advanced_configuration.md#spell-check-("did-you-mean...")) can be configured in `solrconfig.xml`.
> 
> By default, these files are copied from the `fulltextsearch/conf/extras/` directory over to the new index location. In order to use your own files, copy these files into a location of your choosing (for example `mysite/data/solr/`), and tell Solr to use this folder with the `extraspath` [configuration setting](#solr-server-parameters). Run a [`Solr_Configure](#solr-configure) to apply these changes.
> 
> You can also define these on an index-by-index basis by defining `SolrIndex->getExtrasPath()`.

The files are contained in https://github.com/silverstripe/silverstripe-fulltextsearch/tree/3/conf/solr.

Next step for us is probably to validate that those files are being copied appropriately to their expected location in our test project.

Let's also double check what's the process that is meant to make this happen.

**Comment by maxime-rainville** [comment]
I'm thinking maybe we start by manually copying those config files in GIT to their expected location to see if this unstuck the Solr_Config task.

**Comment by sabina-talipova** [comment]
@maxime-rainville, I created new project based on 4.13 release (4.12 didn't work on prod platform last time), copied files that should be copied from fulltextsearch module during task execution. New code base was deployed on two different platforms, but still on prod platform search doesn't work, on uat everything works ok.
Probably, we should keep going to discuss this issue with our platform support team.

**Comment by maxime-rainville** [comment]
Test environement is working again. This should unblock this one.

**Comment by maxime-rainville** [comment]
Try deploying CMS 5 to SC. Double check that Solr Search works.

**Comment by sabina-talipova** [comment]
Initially I've done test on version that was deployed last time and where Solr search didn't work ([CMS 4.13](https://github.com/silverstripeltd/cms-testing-recipes/compare/sc-aws-4.13.0-stable-sink-p81-solr)). Solr search tests passed. 
After that I've deployed CMS 5.0 and also Solr search tests passed.   
I've deployed CMS 5 to both environment SC and SCPS. Solar search still doesn't work on SCPS.
Related ticket for SCPS already exists [Regression: Execution "dev/tasks/Solr_Configure" returns error.](https://app.zenhub.com/workspaces/cms-squad-5d12de441659a92b9fa61677/issues/gh/silverstripe/cwp-search/58)
